walletnode/api/design/artworks.go

```diff
@@ -307,3 +340,190 @@ var RegisterTaskPayload = Type("RegisterTaskPayload", func() {
 	})
 	Required("taskId")
 })
+
+// SearchArtworkParams are query params to searchArtwork request
+var SearchArtworkParams = func() {
+	Attribute("artist", String, func() {
+		Description("Artist PastelID or special value; mine")
+		MaxLength(256)
+	})
+	Attribute("limit", Int, func() {
+		Description("Number of results to be return")
+		Minimum(10)
+		Default(10)
+		Example(10)
+	})
+
+	Attribute("artist_name", String, func() {
+		Description("search for matches in ticket's field - artist_name")
+		MaxLength(256)
+	})
+	Attribute("term", String, func() {
+		Description("search for matches in ticket's field - artwork_title")
+		MaxLength(256)
+	})
+	Attribute("series", String, func() {
+		Description("search for matches in ticket's field - artwork_series_name")
+		MaxLength(256)
+	})
+	Attribute("descr", String, func() {
+		Description("search for matches in ticket's field - artist_written_statement")
+		MaxLength(256)
+	})
+
+	Attribute("keyword", String, func() {
+		Description("search for matches in ticket's field - artwork_keyword_set")
+		MaxLength(256)
+	})
+
+	Attribute("min_block", Int, func() {
+		Description("return art with art min_block <= activation ticket txid ")
+	})
+
+	Attribute("max_block", Int, func() {
```

Comment:
Where validation? Will it return an error if I ask `max_block=-1111`?  please fix it everywhere